tests/integration/templates/apigateway_integration.json

```diff
@@ -152,13 +152,17 @@
     "ApiGatewayUsagePlan": {
       "Type": "AWS::ApiGateway::UsagePlan",
       "Properties": {
-        "Quota": {"Limit" : 123},
+        "Quota": {"Limit" : "123"},
         "ApiStages": [
           {
             "ApiId": {"Ref": "ApiGatewayRestApi"},
             "Stage": {"Ref": "ApiGWStage"}
           }
-        ]
+        ],
+        "Throttle": {
+          "BurstLimit": "500",
```

Comment:
Will get properly converted to `int` on deployment (otherwise we would get a parameter validation error from boto3).